src/build/create-buildinfo.groovy

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
How about https://github.com/alx3apps/jgit-buildnumber plugin? maven-git-commit-id plugin looks too complex and hard to configure.